calico_containers/calico_ctl/node.py

```diff
@@ -241,8 +241,12 @@ def node_start(node_image, runtime, log_dir, ip, ip6, as_num, detach,
 
     # Print warnings for any known system issues before continuing
     using_docker = True if runtime == 'docker' else False
```

Comment:
not you i realise ...  but are we really using a ternary operator to convert a boolean to a boolean?